exec/java-exec/src/main/java/org/apache/drill/exec/expr/fn/impl/StringFunctionHelpers.java

```diff
@@ -169,7 +169,7 @@ public static void initCap(int start, int end, DrillBuf inBuf, DrillBuf outBuf)
         if (currentByte >= 0x30 && currentByte <= 0x39) { // 0-9
           // noop
         } else if (currentByte >= 0x41 && currentByte <= 0x5A) { // A-Z
-          currentByte -= 0x20; // Lowercase this character
+          currentByte += 0x20; // Lowercase this character
```

Comment:
I did not notice any significant difference in performance, so will replace to `Character` methods.